### PR TITLE
improve navbar, context & audio player menu scrolling experience

### PR DESCRIPTION
**Pull request by muhajirdev**
Cannot provide screenshot.
you need to compare the experience directly on your phone

### CODE REVIEW

File: src/components/AudioPlayer/AudioPlayer.module.scss [hunk 1]
```diff
@@ -31,5 +31,5 @@
 }
 
 .containerMinimized.containerDefault {
-  height: constants.$audio-player-minimized-height;
+  transform: translate3d(0, constants.$audio-player-minimized-height, 0);
```
Comment (muhajirdev) [inline src/components/AudioPlayer/AudioPlayer.module.scss:34]: based on some resource, it says translate3d is faster than `translate`.
but I will double check

File: src/components/AudioPlayer/AudioPlayerSlider.module.scss [hunk 2]
```diff
@@ -51,7 +51,6 @@
   width: 100%;
   transform: translate(-50%, 0);
   position: fixed;
-  inset-block-end: constants.$audio-player-minimized-height;
```
Comment (muhajirdev) [inline src/components/AudioPlayer/AudioPlayerSlider.module.scss:54]: the slider was not working on next.quran.com. deleting this fixed the issue

File: src/components/AudioPlayer/AudioPlayer.module.scss (hunk 1 above)
Comment (tifa2UP) [reply src/components/AudioPlayer/AudioPlayer.module.scss:35]: Nice, we should add this comment in the code

File: src/components/AudioPlayer/AudioPlayer.module.scss [hunk 3]
```diff
@@ -24,12 +24,13 @@
   @include breakpoints.tablet {
     height: constants.$audio-player-default-desktop-height;
   }
+  will-change: transform;
```
Comment (tifa2UP) [inline src/components/AudioPlayer/AudioPlayer.module.scss:27]: There are a bunch of warning about will change: https://developer.mozilla.org/en-US/docs/Web/CSS/will-change

I think that using it is still the right call given how frequently this animation happens.

**Review by tifa2UP — APPROVED** [review]

**Pull request merged**